pkg/cmd/cli/describe/deployments.go

```diff
@@ -260,16 +261,98 @@ func printDeploymentConfigSpec(spec deployapi.DeploymentConfigSpec, w io.Writer)
 			spec.Replicas)
 	}
 
-	fmt.Fprintf(w, "  Containers:\n  NAME\tIMAGE\tENV\n")
-	for _, container := range spec.Template.Spec.Containers {
-		fmt.Fprintf(w, "  %s\t%s\t%s\n",
-			container.Name,
-			container.Image,
-			formatLabels(convertEnv(container.Env)))
-	}
+	fmt.Fprintf(w, "  Containers:\n")
+	describeContainers(spec.Template.Spec, w)
+
 	return nil
 }
 
+// TODO: We should reuse this from upstream
+func describeContainers(spec kapi.PodSpec, w io.Writer) {
```

Comment:
The upstream describer needs a pod and returns info about its status. I don't think we can reuse it if we don't refactor it first.